tests/integration/source_code/solacc.py

```diff
@@ -63,17 +75,34 @@ def lint_all():
             parseable += 1
         except Exception as e:  # pylint: disable=broad-except
             # here we're most likely catching astroid & sqlglot errors
-            logger.error(f"Error during parsing of {file}: {e}".replace("\n", " "), exc_info=e)
+            if file_to_lint:
+                logger.error(f"Error during parsing of {file}: {e}".replace("\n", " "), exc_info=e)
+            else:
+                logger.error(f"Error during parsing of {file}: {e}".replace("\n", " "))
```

Comment:
Why are these branches needed? 🤔

I think here we can also use `logger.exception()` which will automatically include the exception stack trace, etc?

Reply:
we only want the stack trace when debugging a single file otherwise it pollutes the output